features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/ProtectedApp.jsx

```diff
@@ -21,9 +21,10 @@ import qs from 'qs';
 import { withTheme } from '@material-ui/core/styles';
```

Comment:
Please fix the following eslint issues.

/home/travis/build/wso2/carbon-apimgt/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/ProtectedApp.jsx
   71:77  error  A space is required before '}'       object-curly-spacing
  180:1   error  More than 2 blank lines not allowed  no-multiple-empty-lines
  222:42  error  Operator '+' must be spaced          space-infix-ops